components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/Nimbus.kt

```diff
@@ -27,15 +27,9 @@ import mozilla.telemetry.glean.Glean
 import org.json.JSONObject
 import org.mozilla.experiments.nimbus.GleanMetrics.NimbusEvents
 import org.mozilla.experiments.nimbus.GleanMetrics.NimbusHealth
-import org.mozilla.experiments.nimbus.internal.AppContext
+import org.mozilla.experiments.nimbus.internal.*
```

Comment:
I don't know why this happened, I'm going to blame Intellij. Happy to revert this if it'd be preferred.

Reply:
ktlint and detekt made it clear I needed to revert it 😂